test/test-daemon.c

```diff
@@ -100,6 +100,97 @@ flush_stream_to_string (GInputStream *stream)
 }
 
 static void
+test_feature_testing_works (DaemonFixture *fixture,
+                            gconstpointer user_data)
+{
+  SoupSession *session;
+  SoupRequestHTTP *req;
+  GInputStream *stream;
+  GError *error = NULL;
+  GString *reply;
+  SoupMessage *message;
+  gchar *req_uri;
+
+  // Test for a known feature.
+  req_uri = g_strdup_printf ("http://localhost:%s/test?feature=query-param-defaultOp",
+                             fixture->port);
+
+  session = soup_session_new ();
+  req = soup_session_request_http (session, SOUP_METHOD_GET,
+                                   req_uri,
+                                   &error);
+  g_assert_no_error (error);
+  g_free (req_uri);
+
+  stream = soup_request_send (SOUP_REQUEST (req), NULL, &error);
+  g_assert_no_error (error);
+  g_assert_nonnull (stream);
+
+  reply = flush_stream_to_string (stream);
+  g_assert_cmpint (reply->len, ==, 0);
+
+  message = soup_request_http_get_message (req);
+  g_assert_cmpint (message->status_code, ==, 200);
+
+  g_string_free (reply, TRUE);
+  g_object_unref (req);
+  g_object_unref (session);
+  g_object_unref (message);
```

Comment:
I just copied this code from `test_get_query_returns_json()`, which doesn't appear to unref `stream`...

How does one unref a `GInputStream`?  `g_object_unref (stream);`?